app/assets/stylesheets/application.css

```diff
@@ -77,12 +82,28 @@
   text-align: right;
 }
 
-.link,
+.link-to-all-post {
+  text-align: center;
+  margin-top: 10px;
+  border: 2px solid black;
+}
+
 .like-form {
   text-align: center;
   margin-top: 10px;
 }
 
+.post-num {
+  text-align: center;
+  margin-top: 10px;
+}
+
+button {
+  padding: 5px;
+  border-radius: 10px;
+  margin-left: 40rem;
+}
+
```

Comment:
- If you faced this issue as it happens on my end, I would recommend using the following [link](https://stackoverflow.com/questions/71538223/tailwind-css-not-being-generated-in-a-rails-7-project-in-heroku) to see the answer that has been mentioned to fix it.

![image](https://user-images.githubusercontent.com/55635977/192633936-9a0d9978-b0cf-4a97-93a9-81afc688c621.png)